### ISSUE TITLE
pointcloud regression tests fail on 32 bit

### ISSUE THREAD

**Issue by df7cb** [pgpointcloud/pointcloud#256]
The regression tests fail on Debian/i386:
```
9.4/regress ...
12:05:51 *** pgsql/regression.diffs ***
12:05:51 + echo *** pgsql/regression.diffs ***
12:05:51 + cat pgsql/regression.diffs
12:05:51 *** /tmp/autopkgtest.efWSUZ/tree/pgsql/expected/pointcloud_pg9.out	2020-01-07 20:42:28.000000000 +0000
12:05:51 --- /tmp/autopkgtest.efWSUZ/tree/pgsql/results/pointcloud_pg9.out	2020-01-08 11:05:49.799218922 +0000
12:05:51 ***************
12:05:51 *** 395,401 ****
12:05:51   SELECT Sum(PC_MemSize(pa)) FROM pa_test_dim;
12:05:51    sum 
12:05:51   -----
12:05:51 !  684
12:05:51   (1 row)
12:05:51   
12:05:51   SELECT Sum(PC_PatchMax(pa,'x')) FROM pa_test_dim;
12:05:51 --- 395,401 ----
12:05:51   SELECT Sum(PC_MemSize(pa)) FROM pa_test_dim;
12:05:51    sum 
12:05:51   -----
12:05:51 !  668
12:05:51   (1 row)
12:05:51   
12:05:51   SELECT Sum(PC_PatchMax(pa,'x')) FROM pa_test_dim;
12:05:51 ***************
12:05:51 *** 431,437 ****
12:05:51   SELECT Sum(PC_MemSize(pa)) FROM pa_test_dim;
12:05:51    sum  
12:05:51   ------
12:05:51 !  8733
12:05:51   (1 row)
12:05:51   
12:05:51   SELECT Max(PC_PatchMax(pa,'x')) FROM pa_test_dim;
12:05:51 --- 431,437 ----
12:05:51   SELECT Sum(PC_MemSize(pa)) FROM pa_test_dim;
12:05:51    sum  
12:05:51   ------
12:05:51 !  8713
12:05:51   (1 row)
12:05:51   
12:05:51   SELECT Max(PC_PatchMax(pa,'x')) FROM pa_test_dim;
```

All Debian/Ubuntu releases are affected.

**Comment by pblottiere** [comment]
Thanks for the report @df7cb, I'm going to take a look asap.

**Comment by pblottiere** [comment]
Hi @df7cb,

32 bit support should be fixed now. Let me know if you have more issue.

And sorry for the delay :/.

Thanks!

**Comment by df7cb** [comment]
Hi,
could you tag a new pgpointcloud release so we get this fix into the Debian package?
Thanks!

**Comment by df7cb** [comment]
Hi,
we need a new release for the Debian package please.

**Comment by pblottiere** [comment]
Hello @df7cb,

I just released a 1.2.2 version with the corresponding fix (one and a half years later, shame on me :/).

**Comment by df7cb** [comment]
Hi @pblottiere, thanks!

Unfortunately, there are more problems: #312, #313.
